apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/grpc/SourceApi.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.api.common.grpc;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import javax.inject.Qualifier;
+
+@Documented
+@Retention(RUNTIME)
+@Qualifier
+public @interface SourceApi {}
```

Comment:
Not sure what to do about this (if anything), but having 2 different `SourceApi`s (one enum in auth, for valid values for our auth; another this tag interface for config) is bit confusing.
Maybe this annotation could be `SourceApiDefinition` or something?

Reply:
OK.

1. Renaming this to `@SourceApiQualifier`, because it is a `Qualifier`
2. Added https://github.com/stargate/stargate/issues/2195 to remove from schema auth

Reply:
Cool, makes sense.